addons/utm/models/utm.py

```diff
@@ -26,6 +27,16 @@ class UtmMedium(models.Model):
     name = fields.Char(string='Medium Name', required=True)
     active = fields.Boolean(default=True)
 
+    @api.ondelete(at_uninstall=False)
+    def _unlink_except_utm_medium_email(self):
```

Comment:
Shouldn't we do ``utm_medium_sms`` for sms mailings ?

Reply:
Yes good idea, I added it.